cmd/grr/main.go

```diff
@@ -94,3 +70,29 @@ func main() {
 		}
 	}
 }
+
+func createRegistry() grizzly.Registry {
+	context, err := config.CurrentContext()
```

Comment:
We want to pass context in rather than pull it from a global. Can you handle this?